src/solver/simulation/economy.cpp

```diff
@@ -165,7 +165,7 @@ bool Economy::year(Progression::Task& progression,
           *pProblemesHebdo[numSpace], state.weekInTheYear, numSpace, hourInTheYear);
 
         // Reinit optimisation if needed
-        pProblemesHebdo[numSpace]->ReinitOptimisation = reinitOptim ? OUI_ANTARES : NON_ANTARES;
+        pProblemesHebdo[numSpace]->ReinitOptimisation = reinitOptim ? true : false;
```

Comment:
```C++
pProblemesHebdo[numSpace]->ReinitOptimisation = reinitOptim;
```
should be enough

Reply:
Moreover, there are code duplication between **economy.cpp** and **adequacy.cpp**.
We should consider removing them.
For instance : 
```C++
catch (Data::UnfeasibleProblemError&)
{
    // need to clean next problemeHebdo
    reinitOptim = true;

    // Indicate failed week list (first week of the year is "week number one" for the
    // user but w=0 for the loop)
    failedWeekList.push_back(w + 1);

    // Define if simulation must be stopped
    if (Data::stopSimulation(study.parameters.include.unfeasibleProblemBehavior))
    {
          return false;
    }
}
```